### PR TITLE
Turn on UPDATE_FRONT_DB_OPTION for all nodes

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Turn on UPDATE_FRONT_DB_OPTION for all nodes

Background:
- This is needed for consistent tx handling in p2p network

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1068

### CODE REVIEW

File: deploy_blockchain_genesis_gcp.sh [hunk 1]
```diff
@@ -329,13 +329,7 @@ for node_index in `seq 0 $(( $NUM_NODES - 1 ))`; do
...
     printf "\n* >> Starting parent node $node_index (${!NODE_TARGET_ADDR}) *********************************************************\n\n"
 
-    if [[ $node_index -ge 5 ]]; then
-        JSON_RPC_OPTION="--json-rpc"
-        UPDATE_FRONT_DB_OPTION="--update-front-db"
-    else
-        JSON_RPC_OPTION=""
-        UPDATE_FRONT_DB_OPTION=""
-    fi
+    UPDATE_FRONT_DB_OPTION="--update-front-db"
```
Comment (liayoo) [inline deploy_blockchain_genesis_gcp.sh:332]: is it ok to remove the JSON_RPC_OPTION as well?

Comment (platfowner) [reply deploy_blockchain_genesis_gcp.sh:337]: Oops, just fixed. PTAL

**Comment by platfowner** [comment]
Comments address and applied to other scripts as well. PTAL~ @liayoo

**Comment by platfowner** [comment]
Thanks for the review. Now merging.. @liayoo

**Pull request merged**